browser/ui/webui/new_tab_page/brave_new_tab_message_handler.cc

```diff
@@ -209,6 +209,11 @@ void BraveNewTabMessageHandler::RegisterMessages() {
       base::BindRepeating(
           &BraveNewTabMessageHandler::HandleBrandedWallpaperLogoClicked,
           base::Unretained(this)));
+  web_ui()->RegisterMessageCallback(
+      "triggerSponsoredRichMediaAdEvent",
```

Comment:
we shouldn't be adding new webui message callbacks. Old callbacks can exist in parallel with mojo based webui

Reply:
cc @aseren

Reply:
Ah, during the first iteration this communication was implemented using mojom. Later I decided to change it to Webui messages because having 2 ways of communication seemed weird to me.
But it's not that hard to return the mojom implementation back.

Reply:
:+1: for using a mojom interface :smile:

Reply:
Fixed